Web/DAL/Entity.Data/Interface/IShopService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Entity.Data.Repository;
+using RockFood.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Entity.Data.Interface
+{
+    public interface IShopService : IDisposable
```

Comment:
it should have methods this logic
seems like u need to cleate FoodService and CustomerService
with its own methods and use CustomerRepository and FoodRepository inside